src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2019,7 +2028,91 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
                 throw new ArgumentNullException(nameof(nuGetProjectContext));
             }
 
+            token.ThrowIfCancellationRequested();
+
+            var results = new List<NuGetProjectAction>();
+
+            // BuildIntegratedNuGetProject type projects are now supports parallel preview action for faster performance.
```

Comment:
I don't think provides much value. It's pretty obvious that it affects multiple projects, hence the name.

Reply:
Removed.